test/replutil.jl

```diff
@@ -20,6 +20,7 @@ method_c1(x::Float64, s::AbstractString...) = true
 buf = IOBuffer()
 Base.show_method_candidates(buf, Base.MethodError(method_c1,(1, 1, "")))
 no_color = "\nClosest candidates are:\n  method_c1(!Matched::Float64, !Matched::AbstractString...)$cfile$c1line"
+@assert length(methods(method_c1)) <= 3
```

Comment:
Wrote that for debugging a test, but actually, it would be useful to leave this here. Will change it to a test and add a comment.